crates/ra_syntax/Cargo.toml

```
@@ -13,7 +13,7 @@ doctest = false
 [dependencies]
 itertools = "0.9.0"
 rowan = "0.9.1"
-rustc_lexer = "0.1.0"
+rustc-ap-rustc_lexer = "652.0.0"
```

```suggestion
rustc_lexer = { version = "652.0.0", package = "rustc-ap-rustc_lexer" }
```

bors r=me otherwise

bors d+